browser/ui/views/tabs/brave_tab_strip_layout.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_TABS_BRAVE_TAB_STRIP_LAYOUT_H_
+#define BRAVE_BROWSER_UI_VIEWS_TABS_BRAVE_TAB_STRIP_LAYOUT_H_
+
+#include <vector>
+
+#include "chrome/browser/ui/views/tabs/tab_strip_layout_types.h"
+#include "chrome/browser/ui/views/tabs/tab_width_constraints.h"
+#include "third_party/abseil-cpp/absl/types/optional.h"
+
+namespace gfx {
+class Rect;
+}  // namespace gfx
+
+std::vector<gfx::Rect> CalculateVerticalTabBounds(
```

Comment:
putting this in brave_tab_strip_layout.h further complicates the circular dependency issues and we should avoid that when possible. There's also no reason to expose this method publicly so I would put it in tab_strip_layout.cc chromium_src override